nodes/number/float_math.py

```diff
@@ -58,6 +66,9 @@ def create(self):
 
     def draw(self, layout):
         layout.prop(self, "operation", text = "")
+        
+    def draw_label(self):
```

Comment:
do we really need an abstraction layer for this? I mean, how much should we abuse labels?
Plus, the current BL way works very well with custom labels (that prevail..), extra things to bother about.
Just wondering if it's worth the effort ..